### ISSUE TITLE
[Feature] Chat interface for better UX

### ISSUE THREAD

**Issue by ombhojane** [veerprakash28/talk-gpt#2]
Problem :
When I entered to the website, I didn't see any chat option or voice activation button on the page. It is hard to interact with the website.

Screenshot : 
![image](https://github.com/veerprakash28/talk-gpt/assets/82753658/421611a2-2b37-472d-8177-786401baa66f)

Proposed Solution : 
![image](https://github.com/veerprakash28/talk-gpt/assets/82753658/adb2cece-d214-4dfa-a1ab-fac878cd30ee)

Let's add a chat interface at the bottom, users can enter text or tap on the voice icon for voice commands.

Please assign this issue to me
Hactoberfest 2023 :)

**Comment by veerprakash28** [comment]
Nice issue. Also, it'll be better if the user has both the option to tap on the start conversation button and say start conversation to start the chat 

I'm assigning this to you. All the best :)

**Comment by veerprakash28** [comment]
You have to do "npm I" where package.json file is present, first locate to that and then install packages

The format for .env is this:
PORT=
apiKey="your openAi key"

**Comment by ombhojane** [comment]
yes, I haven't downloaded the backend server's repo to local, hence error was coming.

Issue is resolved and website working grate! Thanks for guidance :)

**Comment by veerprakash28** [comment]
> yes, I haven't downloaded the backend server's repo to local, hence error was coming.
> 
> Issue is resolved and website working grate! Thanks for guidance :)

Great!

**Comment by ombhojane** [comment]
can you please recheck if the website is working properly or not. https://veerprakash28.github.io/talk-gpt/
When I tried to speak, it not guessed the input

**Comment by veerprakash28** [comment]
I think it stopped working due to some reasons, you can raise and fix this issue if you want. Till then i'll also check and fix this

**Comment by veerprakash28** [comment]
![Screenshot 2023-10-06 at 12 12 32 AM](https://github.com/veerprakash28/talk-gpt/assets/94482800/a5a2368d-d229-494c-89e2-c1ae13691319)

The input is working fine, and for the response my free trial of gpt API is expired, you can try with your api key @ombhojane
